src/metabase/email/messages.clj

```diff
@@ -57,40 +57,46 @@
 
 ;; HACK: temporary workaround to postal requiring a file as the attachment
 (defn- write-byte-array-to-temp-file
-  [img-bytes]
-  (let [file (java.io.File/createTempFile "metabase_pulse_image_" ".png")
-        fos (new java.io.FileOutputStream file)]
-    (.deleteOnExit file)
-    (.write fos img-bytes)
```

Comment:
this was causing a reflection warning, remember you can tag something as a byte array with `^bytes`.